frontend/app/controllers/OffersAndCompetitions.scala

```diff
@@ -4,11 +4,17 @@ import model.ContentItemOffer
 import play.api.mvc.Controller
 import services.GuardianContentService
 
-
 trait OffersAndCompetitions extends Controller {
-  // TODO move this to CachedAction once this work is ready to go into the wild
-  def list = GoogleAuthenticatedStaffAction { implicit request =>
+  def list = CachedAction { implicit request =>
+    /**
+     * Only show items that:
+     * - Don't have the membershipAccess field set
+     * - Have an available image
+     */
```

Comment:
Don't think you really need to do anything but this comment seems fairly redundant here given the code below